.github/workflows/Unit-Test.yml

```diff
@@ -11,6 +11,18 @@ jobs:
         shell: bash
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
+        # Only test relevant files that have changed.
+      - name: Get all changed package files
+        id: changed-ruby-files
+        uses: tj-actions/changed-files@v44
```

Comment:
We could do this in Handoff.yml and pass it here as a variable

Reply:
I've got something pretty close to that in https://github.com/Zopolis4/chromebrew/blob/ed7412a4c41a65fd534c7914d5da1469a8daa313/.github/workflows/Handoff.yml

Reply:
I'm fine with using your changes instead of mine.  As long as it speeds it up.  Currently, the tests take 10 min to run.

Reply:
The unit tests only need to check for changes to the package files and nothing else.